kitipy/executor.py

```diff
@@ -449,6 +449,17 @@ def _remote(
         returncode = sin.channel.recv_exit_status()
         return subprocess.CompletedProcess(cmd, returncode, stdout, stderr)
 
+    def _wrap_cmd(self, cmd: str) -> str:
```

Comment:
Mh I'm wondering if this wouldn't be better placed in a separate class inheriting from `ProxyExecutor` because with this implementation we bind the Executor to bash details and as such it becomes less generic. Or maybe using the dispatcher :thinking: I'm not really sure how this should be implemented :confused:

Reply:
Maybe we can try to expose a more generic API and just support bash/sh for now and see later for other shell like fish or ksh.